bundler/lib/bundler/cli/info.rb

```diff
@@ -62,8 +62,22 @@ def print_gem_info(spec)
       gem_info << "\tBug Tracker: #{metadata["bug_tracker_uri"]}\n" if metadata.key?("bug_tracker_uri")
       gem_info << "\tMailing List: #{metadata["mailing_list_uri"]}\n" if metadata.key?("mailing_list_uri")
       gem_info << "\tPath: #{spec.full_gem_path}\n"
-      gem_info << "\tDefault Gem: yes" if spec.respond_to?(:default_gem?) && spec.default_gem?
+      gem_info << "\tDefault Gem: yes\n" if spec.respond_to?(:default_gem?) && spec.default_gem?
+      gem_info << "\tDependencies:\n"
```

Comment:
`Dependencies` doesn't seem like the right word to me, we're showing what gems are depending on this gem instead of what gems the given gem name depends on, so  `Reverse Dependencies` would be more accurate.